synfig-studio/plugins/lottie-exporter/shapes/gFill.py

```diff
@@ -12,9 +12,83 @@
 from common.Gradient import Gradient
 sys.path.append("..")
 
+
+def gen_radial_gradient(lottie, layer, idx):
+    """
+    Generates the dictionary correspnding to shapes/gFill.json but with radial gradient
+
+    Args:
+    """
+    index = Count()
+    lottie["ty"] = "gf"
+    lottie["r"] = 1    # Don't know it's meaning yet, but works without this also
+    lottie["o"] = {}   # Opacity of the gradient layer
+    lottie["nm"] = layer.get_description()
+    lottie["t"] = 2    # 2 means radial gradient layer
+    lottie["s"] = {}   # Starting point of gradient
+    lottie["e"] = {}   # Ending point of gradient
+    lottie["g"] = {}   # Gradient information is stored here
+
+    # Color Opacity
+    opacity = layer.get_param("amount").get()
+    is_animate = is_animated(opacity[0])
+    if is_animate == 2:
```

Comment:
Maybe change "2" to constant? Just for a better understanding of what is happening.

Reply:
Actually, this "2" constant has been used by me at many more places. Like here: 

- https://github.com/synfig/synfig/blob/4c519a9539905788b66101b4b440880949733a6b/synfig-studio/plugins/lottie-exporter/shapes/star.py#L53

- https://github.com/synfig/synfig/blob/4c519a9539905788b66101b4b440880949733a6b/synfig-studio/plugins/lottie-exporter/shapes/star.py#L72
- https://github.com/synfig/synfig/blob/4c519a9539905788b66101b4b440880949733a6b/synfig-studio/plugins/lottie-exporter/shapes/rectangle.py#L34
- And many more places.

@ice0 , may be we can have a seperate PR to deal with this? ^__^

Reply:
Sure.